napari/layers/points/points.py

```diff
@@ -99,12 +99,6 @@ class Points(Layer):
         of the specified property that are mapped to 0 and 1, respectively.
         The default value is None. If set the none, the clims will be set to
         (property.min(), property.max())
-    out_of_slice_display : bool
-        If True, renders points not just in central plane but also slightly out of slice
-        according to specified point marker size.
-    n_dimensional : bool
-        This property will soon be deprecated in favor of 'out_of_slice_display'.
-        Use that instead.
```

Comment:
Am I right in saying this change means that layers cannot independently set `out_of_slice_display` because we only have one dims model for the whole views?

Reply:
Not quite: layers CAN set the sclicing mode idependently (by choosing the `Projection` mode). However, the *thickness* of the dims model is the same for everything. So you can project the image but not the points, but you can't project different thicknesses for the two.

Reply:
okay interesting - I'll have to play with it!